spring-cloud-gcp-pubsub/src/test/java/org/springframework/cloud/gcp/pubsub/integration/inbound/PubSubInboundChannelAdapterTests.java

```diff
@@ -123,10 +123,7 @@ public void testAckModeAutoAck_nacksWhenDownstreamProcessingFails() {
 		when(this.mockMessageChannel.send(any())).thenThrow(new RuntimeException(EXCEPTION_MESSAGE));
 
 		this.adapter.setAckMode(AckMode.AUTO_ACK);
-
-		assertThatThrownBy(() -> {
-			this.adapter.start();
-		}).hasMessageContaining(EXCEPTION_MESSAGE);
+		this.adapter.start();
```

Comment:
Can you change the name of this method to reflect intended behavior? 

But I am also confused why the exception is no longer thrown. Code under test rethrows in AUTO_ACK mode.

Reply:
I reverted this part of the test since we decided we will implement this fix for the clear regression in this PR and then decide on how exception should be thrown in the next PR.